src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -20,16 +20,21 @@
 {%- endblock %}
 
 {% block money_widget -%}
-    <div class="input-group">
-        {% set append = money_pattern starts with '{{' %}
-        {% if not append %}
-            <span class="input-group-addon">{{ money_pattern|replace({ '{{ widget }}':''}) }}</span>
-        {% endif %}
+    {% set append_before = not (money_pattern starts with '{{') %}
```

Comment:
what about a check like `money_pattern is '{{ widget }}'` wouldnt that be more concise?

Reply:
Don't know, a `starts with` or `ends with` test would still be required in case the currency symbol must be appended.

Reply:
Right, lets call those variables `append` + `prepend`?

Reply:
Done.